index.html

```diff
@@ -3,9 +3,53 @@
   <head>
     <title>Relaxr</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
   </head>
+
   <body>
-    <header>
-    </header>
+ <section class="hero">
+    <header>Relaxr</header>
+    <div id="sub">Get piece of mind with a single tap</div>
+    <div id="get">Get it Now</div>
+    </section>
+
+ <section class="benefits">
+ 	<h2>Benefits</h2>
+ 	<p id= "para1">The perfect personal assistant. Relaxr does your job for you so can enjoy life as it is meant to <br>be<br /></p>
```

Comment:
The space after the equal sign is a typo. I'm actually surprised this works but Chrome will do a lot to fix our errors.